modules/cnv-vm-console-web.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-accessing-vm-consoles.adoc
+
+[id="cnv_vm_console_web-{context}"]
+= Virtual machine console sessions
+
+You can connect to the VNC and serial consoles of a running virtual machine from the *Consoles* tab in the *Virtual Machine Details* screen of the web console.
+
+There are two consoles available: the graphical *VNC Console* and the *Serial Console*. The *VNC Console* opens by default whenever you navigate to the *Consoles* tab. You can switch between the consoles using the xref:cnv-vm-vnc-console-web[*VNC Console*]|xref:cnv-vm-serial-console-web[*Serial Console*] drop-down list.
```

Comment:
Will have to check those xrefs at the door, I'm afraid

Reply:
They should be gone already.